### PR TITLE
 macOS: fix back button for nested nav pages and CI expansion

### PR DESCRIPTION
**Pull request by aarani**

### CODE REVIEW

File: DevopsNuget.config [hunk 1]
```diff
@@ -5,7 +5,6 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
-    <add key="nuget" value="../s/Nuget" />
```
Comment (knocte) [inline DevopsNuget.config:8]: is this related to the PR?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -16,31 +17,37 @@ jobs:
...
 
         If ($Env:GITHUB_REF -eq $Env:DEFAULT_REF_TO_NUGET_PUSH) {
             If ('${{ secrets.NUGET_API_KEY }}' -ne '') {
-                $fileName= "DotNetForms.Platform.GTK.$version.nupkg"
-                nuget push $fileName -ApiKey ${{secrets.NUGET_API_KEY}} -Source https://api.nuget.org/v3/index.json
+                nuget push "DotNetForms.$version.nupkg" -ApiKey ${{secrets.NUGET_API_KEY}} -Source https://api.nuget.org/v3/index.json    
+                nuget push "DotNetForms.Platform.GTK.$version.nupkg" -ApiKey ${{secrets.NUGET_API_KEY}} -Source https://api.nuget.org/v3/index.json
             }
         }
+    - name: Upload packages
```
Comment (knocte) [inline .github/workflows/main.yml]: rename from 'Upload packages' to 'Upload packages as CI artifacts' to not confuse with nuget upload

**Comment by knocte** [comment]
The CI commit message has the what but not the why.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -2,11 +2,12 @@ name: ci
 on: [push, pull_request]
 
 jobs:
-  build_and_deploy:
+  build_and_pack:
```
Comment (knocte) [inline .github/workflows/main.yml]: why rename this?

**Review by knocte — CHANGES_REQUESTED** [review]

File: DevopsNuget.config (hunk 1 above)
Comment (aarani) [reply DevopsNuget.config:8]: yeah the build fails because that folder doesn't exist on our CI

File: .github/workflows/main.yml (hunk 3 above)
Comment (aarani) [reply .github/workflows/main.yml]: should be reverted now

File: DevopsNuget.config (hunk 1 above)
Comment (knocte) [reply DevopsNuget.config:8]: Say this in the commit message then.

**Comment by knocte** [comment]
Superseded by https://github.com/nblockchain/DotNetForms/pull/3

**Pull request closed without merging**